samples/cpp/speculative_decoding_lm/speculative_decoding_lm.cpp

```diff
@@ -33,7 +33,8 @@ int main(int argc, char* argv[]) try {
         main_model_path,
         main_device,
         ov::genai::draft_model(draft_model_path, draft_device),
-        ov::genai::scheduler_config(scheduler_config));
+        ov::genai::scheduler_config(scheduler_config)
+    );
```

Comment:
please, revert this change as it creates merge conflicts with https://github.com/openvinotoolkit/openvino.genai/pull/1364 where this lone is removed.